ui/raidboss/data/05-shb/raid/e1n.js

```diff
@@ -108,37 +84,7 @@
       regexJa: Regexes.startsUsing({ id: '3DA1', source: 'エデン・プライム' }),
       regexCn: Regexes.startsUsing({ id: '3DA1', source: '至尊伊甸' }),
       regexKo: Regexes.startsUsing({ id: '3DA1', source: '에덴 프라임' }),
-      alertText: function(data, matches) {
-        if (matches.target == data.me) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            cn: '死刑点名',
-            ko: '나에게 탱버',
-          };
-        }
-        if (data.role == 'healer') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Tankbuster auf ' + data.ShortName(matches.target),
-            fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            cn: '死刑 -> ' + data.ShortName(matches.target),
-            ko: '탱버 -> ' + data.ShortName(matches.target),
-          };
-        }
-      },
-      infoText: function(data, matches) {
-        if (matches.target != data.me && data.role == 'tank') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Tankbuster auf ' + data.ShortName(matches.target),
-            fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            cn: '死刑 -> ' + data.ShortName(matches.target),
-            ko: '탱버 -> ' + data.ShortName(matches.target),
-          };
-        }
-      },
+      response: Responses.tankBuster(),
```

Comment:
Could you add a condition here of "target is you or role is healer or role is tank" to preserve the previous logic?